apps/apple/package.json

```diff
@@ -21,10 +21,12 @@
     "babel-jest": "^24.9.0",
     "eslint": "^6.5.1",
     "jest": "^24.9.0",
-    "metro-react-native-babel-preset": "^0.58.0",
-    "react-test-renderer": "~16.11.0"
+    "metro-react-native-babel-preset": "^0.58.0"
   },
   "jest": {
     "preset": "react-native"
+  },
+  "workspaces": {
+    "nohoist": ["react-native", "react-native/**", "@babel/runtime/**"]
```

Comment:
You should be able to work around the hoisting of these, but you can do this as a separate step.  We are bundling win32 and windows with react native and react-native-windows hoisted to root.

Reply:
Thanks, was hitting some errors without doing so but I'm sure there's a good workaround. Will followup with you on how win does it